onedal/datatypes/utils/dtype_conversions.cpp

```diff
@@ -0,0 +1,129 @@
+/*******************************************************************************
+* Copyright 2024 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include <unordered_map>
+
+#include <pybind11/pybind11.h>
+
+#include "oneapi/dal/common.hpp"
+#include "oneapi/dal/detail/common.hpp"
+
+#include "onedal/datatypes/utils/common.hpp"
+#include "onedal/datatypes/utils/dtype_conversions.hpp"
+#include "onedal/datatypes/utils/dtype_dispatcher.hpp"
+
+namespace oneapi::dal::python {
+
+using fwd_map_t = std::unordered_map<std::string, dal::data_type>;
```

Comment:
These maps provide translation for dtypes? forward is then implied towards dal, inverse towards sua?  Comments/ documentation would be helpful.

Reply:
Done